test/unit/migrations/migrator-test.js

```diff
@@ -35,6 +37,23 @@ const firstTimeState = {
   meta: { version: 0 },
   data: require('../../../app/scripts/first-time-state'),
 }
+describe('liveMigrations require list', () => {
+  it('should include all the migrations', async () => {
+    const fileNames = await pify(cb => fs.readdir('./app/scripts/migrations/', cb))()
```

Comment:
Instead of using `pify` here, you could use the Promise-returning version of `fs.readdir` instead:
`fs.promises.readdir`. Added in Node.js v10

Reply:
We can do this in a new PR